kie-osgi/kie-karaf-itests/src/test/java/org/kie/karaf/itest/blueprint/KieBlueprintjBPMPersistenceKarafIntegrationTest.java

```diff
@@ -122,6 +122,7 @@ public void testStartProcessEnv() throws Exception {
                                                "javax.sql," +
                                                "org.apache.commons.dbcp2.managed," +
                                                "org.h2")
+                .set(Constants.DYNAMICIMPORT_PACKAGE, "*")
```

Comment:
Don't we need this line also on main branch ? @grgrzybek @jiripetrlik ?

Reply:
true - it's needed to use different JDBC driver than H2.